server/src/main/java/io/littlehorse/server/KafkaStreamsServerImpl.java

```diff
@@ -39,166 +29,24 @@
-import io.littlehorse.common.model.metadatacommand.subcommand.DeleteWfSpecRequestModel;
-import io.littlehorse.common.model.metadatacommand.subcommand.MigrateWfSpecRequestModel;
-import io.littlehorse.common.model.metadatacommand.subcommand.PutExternalEventDefRequestModel;
-import io.littlehorse.common.model.metadatacommand.subcommand.PutPrincipalRequestModel;
-import io.littlehorse.common.model.metadatacommand.subcommand.PutTaskDefRequestModel;
-import io.littlehorse.common.model.metadatacommand.subcommand.PutTenantRequestModel;
-import io.littlehorse.common.model.metadatacommand.subcommand.PutUserTaskDefRequestModel;
-import io.littlehorse.common.model.metadatacommand.subcommand.PutWfSpecRequestModel;
-import io.littlehorse.common.model.metadatacommand.subcommand.PutWorkflowEventDefRequestModel;
+import io.littlehorse.common.model.metadatacommand.subcommand.*;
 import io.littlehorse.common.proto.InternalScanResponse;
 import io.littlehorse.common.proto.WaitForCommandResponse;
 import io.littlehorse.common.util.LHProducer;
 import io.littlehorse.common.util.LHUtil;
-import io.littlehorse.sdk.common.proto.ACLAction;
-import io.littlehorse.sdk.common.proto.ACLResource;
-import io.littlehorse.sdk.common.proto.AssignUserTaskRunRequest;
-import io.littlehorse.sdk.common.proto.AwaitWorkflowEventRequest;
-import io.littlehorse.sdk.common.proto.CancelUserTaskRunRequest;
-import io.littlehorse.sdk.common.proto.CompleteUserTaskRunRequest;
-import io.littlehorse.sdk.common.proto.DeleteExternalEventDefRequest;
-import io.littlehorse.sdk.common.proto.DeleteTaskDefRequest;
-import io.littlehorse.sdk.common.proto.DeleteUserTaskDefRequest;
-import io.littlehorse.sdk.common.proto.DeleteWfRunRequest;
-import io.littlehorse.sdk.common.proto.DeleteWfSpecRequest;
-import io.littlehorse.sdk.common.proto.ExternalEvent;
-import io.littlehorse.sdk.common.proto.ExternalEventDef;
-import io.littlehorse.sdk.common.proto.ExternalEventDefId;
-import io.littlehorse.sdk.common.proto.ExternalEventDefIdList;
-import io.littlehorse.sdk.common.proto.ExternalEventId;
-import io.littlehorse.sdk.common.proto.ExternalEventIdList;
-import io.littlehorse.sdk.common.proto.ExternalEventList;
-import io.littlehorse.sdk.common.proto.GetLatestUserTaskDefRequest;
-import io.littlehorse.sdk.common.proto.GetLatestWfSpecRequest;
-import io.littlehorse.sdk.common.proto.LHHostInfo;
-import io.littlehorse.sdk.common.proto.ListExternalEventsRequest;
-import io.littlehorse.sdk.common.proto.ListNodeRunsRequest;
-import io.littlehorse.sdk.common.proto.ListTaskMetricsRequest;
-import io.littlehorse.sdk.common.proto.ListTaskMetricsResponse;
-import io.littlehorse.sdk.common.proto.ListTaskRunsRequest;
-import io.littlehorse.sdk.common.proto.ListUserTaskRunRequest;
-import io.littlehorse.sdk.common.proto.ListVariablesRequest;
-import io.littlehorse.sdk.common.proto.ListWfMetricsRequest;
-import io.littlehorse.sdk.common.proto.ListWfMetricsResponse;
+import io.littlehorse.sdk.common.proto.*;
```

Comment:
Hmm, @eduwercamacaro do we have a style guide for these changes?

Reply:
spotless only verify linting problems. In order to add a validation for this can of changes we need a tool like Checkstyle. see ([AvoidStarImport](https://checkstyle.sourceforge.io/checks/imports/avoidstarimport.html#AvoidStarImport))

Reply:
Do you have an opinion about these changes?

Reply:
we should not use wildcard imports